docs/source/torch.compiler.rst

```diff
@@ -64,9 +66,9 @@ Some of the most commonly used backends include:
    * - Backend
      - Description
    * - ``torch.compile(m, backend="onnxrt")``
-     - Uses ONNXRT for inference on CPU/GPU. `Read more <https://onnxruntime.ai/>`__
```

Comment:
Remove from inference only?

Reply:
IIUC @abock asked to put them in both inference and training. @abock and @wschin what do you advice?

Reply:
DORT supports both training and inference. DORT's major value is training (inference is exporter's responsibility, ideally. If exporter fails, DORT is considered a backup solution), so let's remove `inference only`.

Reply:
ok, removed DORT from inference